src/pages/video-test.tsx

```diff
@@ -106,12 +123,21 @@ const EggheadPlayer: React.FC<{videoResource: VideoResource}> = ({
           </div>
           <div className="lg:col-span-3 side-bar">
             <div className="relative h-full">
-              <Tabs className="max-h-[500px] lg:max-h-[none] lg:absolute left-0 top-0 w-full h-full flex flex-col bg-gray-100 dark:bg-gray-1000 text-gray-900 dark:text-white">
+              <Tabs
+                index={playerPrefs.sideBar?.activeTab || 0}
+                onChange={(tabIndex) =>
+                  playerPrefs.setPlayerPrefs({sideBar: {activeTab: tabIndex}})
+                }
```

Comment:
Get and set active tab using `usePlayerPrefs` based its index as that's what ReachUI Tabs expect.